addons/website_forum/static/src/js/website_forum.js

```diff
@@ -501,7 +502,7 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
      */
     _onCloseIntroClick: function (ev) {
         ev.preventDefault();
-        document.cookie = 'forum_welcome_message = false';
+        set_cookie('forum_welcome_message', false, 'optional');
```

Comment:
Missing `ttl` value so the `optional` is not kept. This makes me think, wouldn't it make sense to check the type of `ttl` in `set_cookie` (detect if not falsy or integer) to avoid similar issues at other places ?